spring-cloud-stream-binder-test/src/main/java/org/springframework/cloud/stream/binder/AbstractBinderTests.java

```diff
@@ -121,36 +129,52 @@ public void testClean() throws Exception {
 				.isRunning()).isFalse();
 	}
 
+	@SuppressWarnings("rawtypes")
 	@Test
 	public void testSendAndReceive() throws Exception {
 		Binder binder = getBinder();
-		BindingProperties outputBindingProperties = createProducerBindingProperties(
-				createProducerProperties());
-		DirectChannel moduleOutputChannel = createBindableChannel("output",
-				outputBindingProperties);
-		QueueChannel moduleInputChannel = new QueueChannel();
-		Binding<MessageChannel> producerBinding = binder.bindProducer("foo.0",
-				moduleOutputChannel, outputBindingProperties.getProducer());
-		Binding<MessageChannel> consumerBinding = binder.bindConsumer("foo.0",
-				"testSendAndReceive", moduleInputChannel, createConsumerProperties());
-		// Bypass conversion we are only testing sendReceive
-		Message<?> message = MessageBuilder.withPayload("foo".getBytes())
-				.setHeader(MessageHeaders.CONTENT_TYPE,
-						MimeTypeUtils.APPLICATION_OCTET_STREAM)
+		BindingProperties outputBindingProperties = createProducerBindingProperties(createProducerProperties());
```

Comment:
I simply __hate__ reformatting being done as part of a big PR like this - it detracts from the meaningful changes and makes the review much harder - I __strongly__ believe such efforts (while worthwhile) __must__ be done as a separate PR.

Reply:
I agree Gary and that was an over site on my part.

Reply:
Thanks; turned out to be not too bad; just that this was the first file and I thought uh-oh 😄